### PR TITLE
Add NYU School of Engineering CS dept

### PR DESCRIPTION
**Pull request by moyix**
I've removed the ECE faculty from this pull request. The remaining ones are all CS.

### CODE REVIEW

File: faculty-affiliations.csv [hunk 1]
```diff
@@ -774,14 +774,34 @@ Panayotis Mavromatis , New York University
 Patrick Cousot , New York University
 Ralph Grishman , New York University
 Richard Bonneau , New York University
-Richard Cole 0001, New York University
+Richard Cole , New York University
```
Comment (emeryberger) [inline faculty-affiliations.csv:777]: This is incorrect: AFAICT the correct DBLP entry for Richard Cole is in fact Richard Cole 0001 (see http://dblp.uni-trier.de/pers/hd/c/Cole_0001:Richard).

File: faculty-affiliations.csv [hunk 2]
```diff
@@ -774,14 +774,34 @@ Panayotis Mavromatis , New York University
...
 Rob Fergus , New York University
 Robert Grimm , New York University
```
Comment (emeryberger) [inline faculty-affiliations.csv:779]: Actually should be removed (no longer on faculty page, moved to Goldman-Sachs).

**Comment by emeryberger** [comment]
I added some comments but at a glance, I see a number of names not listed here: http://www.cs.nyu.edu/dynamic/people/faculty/type/1/

**Comment by emeryberger** [comment]
I just added another comment; closing for now since this is not NYU CS.

**Comment by moyix** [comment]
You are looking in the wrong place.

http://engineering.nyu.edu/academics/departments/computer-science-engineering/people#faculty

Are you excluding CS departments that reside inside engineering schools?

**Comment by emeryberger** [comment]
Only including CS, EECS or equivalent departments. This looks like the moral equivalent of what most universities call an ECE department?

**Comment by moyix** [comment]
Nope, the school of engineering also has an ECE department:

http://engineering.nyu.edu/academics/departments/electrical/people#core

The current situation is due to the fact that the school of engineering was a separate university (Polytechnic University) until a few years ago, and the departments have not yet merged. So there are duplicates of a few departments, including CS.

**Comment by moyix** [comment]
Hmm, I also noticed that two faculty from NYU SoE CSE are actually already included (Juliana Freire and Guido Gerig). So it seems like the rest of the department should be included too, no?

**Comment by emeryberger** [comment]
If they were incorrectly included, for now, they should be removed. This NYU situation is highly unusual (as you are no doubt aware), so it's not yet clear to me how to manage it.

**Comment by moyix** [comment]
No worries. I do urge you to consider including engineering depts as well though. Particularly in more applied areas like security people who do very CS-y things like software security will still end up in ECE. Examples include [David Brumley](https://users.ece.cmu.edu/~dbrumley/) and [Manuel Egele](https://www.bu.edu/ece/people/faculty/a-g/manuel-egele/).

Thanks for putting in the work on this! I think it'll be a great resource (I did the same thing by hand for security when I was looking at grad schools back in 2007). I hope I didn't come off as rude in this thread, it's just frustrating to see one's work and colleagues disqualified from a ranking like this.

**Comment by emeryberger** [comment]
Sure, no problem. I have been discussing how best to handle this with several folks to come up with a reasonable, uniform policy that will actually make sense to graduate students.

**Comment by emeryberger** [comment]
Please make a new PR that only includes the faculty from Polytechnic - thanks.

**Pull request closed without merging**